### PR TITLE
feat(IFO): Show CAKE spent info and tooltip

### PR DESCRIPTION
**Pull request by Woody810**
Live:

Finished:

ToolTip content:

### CODE REVIEW

**Review by Chef-Yogi — DISMISSED** [review]

File: src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx [hunk 1]
```diff
@@ -256,6 +257,18 @@ const IfoCardTokens: React.FC<React.PropsWithChildren<IfoCardTokensProps>> = ({
               userAmount={userPoolCharacteristics.amountTokenCommittedInLP}
               totalAmount={publicPoolCharacteristics.totalAmountPool}
             />
+            <Flex>
+              <Box>
+                <Label>{t('Your %symbol% refunds', { symbol: currency.symbol })}</Label>
+                <BalanceWithLoading
+                  bold
+                  prefix="~"
```
Comment (ChefMomota) [inline src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx:265]: Why should we show `~`?

File: src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoVestingCard/TotalPurchased.tsx [hunk 2]
```diff
@@ -35,6 +35,34 @@ const TotalPurchased: React.FC<React.PropsWithChildren<TotalPurchasedProps>> = (
...
+        </Box>
+      </Flex>
+      <Flex ml="48px">
+        <Box>
+          <Text bold color="secondary" fontSize="12px" textTransform="uppercase">
+            {t('Your %symbol% refunds', { symbol: ifo.currency.symbol })}
+          </Text>
+          <BalanceWithLoading
+            bold
+            prefix="~"
```
Comment (ChefMomota) [inline src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoVestingCard/TotalPurchased.tsx:59]: remove `prefix`

File: src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx (hunk 1 above)
Comment (ChefMomota) [inline src/views/Ifos/components/IfoFoldableCard/IfoPoolCard/IfoCardTokens.tsx:265]: remove `prefix`

**Comment by ChefMomota** [comment]
IMO, live no need to show refund amount

**Comment by Woody810** [comment]
> IMO, live no need to show refund amount

Users often complain that there is no real-time refund information on live, which will determine whether they want to increase the CAKE invested.

**Review by chef-jojo — APPROVED** [review]

**Pull request merged**